src/Rules/PoliteUsername.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ARKEcosystem\Fortify\Rules;
+
+use Illuminate\Contracts\Validation\Rule;
+use Snipe\BanBuilder\CensorWords;
+
+final class PoliteUsername implements Rule
+{
+    public function __construct(public CensorWords $censor)
+    {
+        $this->censor->badwords = config('profanities.'.config('app.locale'));
```

Comment:
can we make it a single list without relying on locale? otherwise if we focus on english only then people will start registering `puta` and we don't care as that's in another word list